data_engineering/de_intro/intro.py

```diff
@@ -0,0 +1,70 @@
+"""This module contains execution exercise DE Intro Task"""
+import glob
+import os
+import typing as t
+
+import pandas as pd
+from pandas.core.frame import DataFrame
+
+
+def download_file(s3_connect, bucket_name: str) -> t.Optional[str]:
```

Comment:
no need to download files locally, it's better to read file right from object storage